sjsonnet/src/sjsonnet/Util.scala

```diff
@@ -33,4 +33,20 @@ object Util{
         new String(range.dropWhile(_ < 0).takeWhile(_ < s.length).map(s).toArray)
     }
   }
+
+  val isWindows: Boolean = System.getProperty("os.name").toLowerCase.startsWith("windows")
```

Comment:
This matches https://github.com/databricks/sjsonnet/blob/759cea713bc8e39f45b844a75358fd780f75d480/sjsonnet/client/src/sjsonnet/client/Util.java#L11, but we can't directly that other implementation here.

Reply:
Come to think of it, though, I'm wondering whether this should be lifted to `Platform` to ensure it won't blow up on Scala JS.

Reply:
Yep, we need to wrap in options to avoid an NPE under ScalaJS: https://github.com/databricks/sjsonnet/pull/215/commits/d745f75eeb855f9d57216387ab7c32c177776054

See https://scastie.scala-lang.org/VuigbI2gS5G1WOnTYCdTdg for seeing some default properties in ScalaJS (look at Javascript console output after running).